nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,37 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
+    pkgs.symlinkJoin {
+      name = "firejail-" + pkg.name;
+      paths = [ pkg ];
+      buildInputs = with pkgs; [ tree ];
+      postBuild = ''
+        for bin in $(find "$out/bin" -type l); do
+        oldbin="$(readlink "$bin")" 
+        rm "$bin"
+        cat <<_EOF >"$bin"
```

Comment:
Instead of manually creating shell scripts (they are admittedly basic), just use makeWrapper.

```
  makeWrapper $bin /run/wrappers/bin/firejail \
    --argv0 $bin \
    --add-flags $oldbin
```

Reply:
I tried this before analyzing it and it failed with `mkdir: cannot create directory '/run': Permission denied` because the second argument to makeWrapper is the destination it tries to create. I don't see how makeWrapper could be made to work here.